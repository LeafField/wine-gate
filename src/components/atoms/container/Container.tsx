import React, { PropsWithChildren, FC } from "react";

const Container: FC<PropsWithChildren> = ({ children }) => {
  return (
    <div className=" mx-auto grid  justify-center gap-x-5 px-6 grid-areas empty-hero tablet:px-0">
      {children}
    </div>
  );
};

export default Container;
