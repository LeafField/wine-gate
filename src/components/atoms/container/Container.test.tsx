import { screen, render } from "@testing-library/react";
import Container from "./Container";

describe("Contaierの単体テスト", () => {
  test("スナップショットテスト", () => {
    const { container } = render(<Container />);
    expect(container).toMatchSnapshot();
  });
});
