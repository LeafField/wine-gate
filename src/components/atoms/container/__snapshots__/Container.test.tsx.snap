// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contaierの単体テスト スナップショットテスト 1`] = `
<div>
  <div
    class="grid-areas empty-hero mx-auto grid max-w-[1440px] grid-cols-[1039px_333px] justify-center gap-x-5 px-6 tablet:px-0"
  />
</div>
`;
